=============== positive tests =====================
Exception: 'func' called dynamically but is not __DynamicallyCallable
Exception: 'async_func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_async_func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_async_func' called dynamically but is not __DynamicallyCallable
Exception: 'A::func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_func' called dynamically but is not __DynamicallyCallable
Exception: 'A::async_func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_async_func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_async_func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_async_func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_func' called dynamically but is not __DynamicallyCallable
Exception: 'B::static_func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_async_func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_async_func' called dynamically but is not __DynamicallyCallable
Exception: 'B::static_async_func' called dynamically but is not __DynamicallyCallable
Exception: 'A' constructed dynamically
Exception: 'A::func' called dynamically but is not __DynamicallyCallable
Exception: 'A::async_func' called dynamically but is not __DynamicallyCallable
Exception: 'func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_func' called dynamically but is not __DynamicallyCallable
Exception: 'A::func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_func' called dynamically but is not __DynamicallyCallable
Exception: 'func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_func' called dynamically but is not __DynamicallyCallable
Exception: 'A::func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_func' called dynamically but is not __DynamicallyCallable
Exception: 'func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_func' called dynamically but is not __DynamicallyCallable
Exception: 'A::func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_func' called dynamically but is not __DynamicallyCallable
Exception: 'func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_func' called dynamically but is not __DynamicallyCallable
Exception: 'A::func' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_func' called dynamically but is not __DynamicallyCallable
Exception: 'cmp' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_cmp' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_cmp' called dynamically but is not __DynamicallyCallable
Exception: 'A::cmp' called dynamically but is not __DynamicallyCallable
Exception: 'A::static_cmp' called dynamically but is not __DynamicallyCallable
=============== negative tests =====================
Exception: 'count' called dynamically without using a function pointer
Exception: 'count' called dynamically without using a function pointer
Exception: 'count' called dynamically without using a function pointer
Exception: 'count' called dynamically without using a function pointer
Exception: 'count' called dynamically without using a function pointer
Exception: 'HH\Vector::fromItems' called dynamically without using a function pointer
Exception: 'HH\Vector::fromItems' called dynamically without using a function pointer
Exception: 'HH\Vector::fromItems' called dynamically without using a function pointer
Exception: 'HH\Vector::fromItems' called dynamically without using a function pointer
Exception: 'HH\Vector::firstValue' called dynamically without using a function pointer
Exception: 'HH\Vector::fromItems' called dynamically without using a function pointer
Exception: 'HH\Vector::firstValue' called dynamically without using a function pointer
Exception: 'HH\Vector::fromItems' called dynamically without using a function pointer
Exception: 'HH\Vector::fromItems' called dynamically without using a function pointer
Exception: 'HH\Vector::fromItems' called dynamically without using a function pointer
Exception: 'HH\Vector::fromItems' called dynamically without using a function pointer
Exception: 'HH\Vector::fromItems' called dynamically without using a function pointer
Exception: 'HH\Vector::fromItems' called dynamically without using a function pointer
Exception: 'HH\Vector::fromItems' called dynamically without using a function pointer
Exception: 'HH\Vector::fromItems' called dynamically without using a function pointer
Exception: 'HH\Vector::firstValue' called dynamically without using a function pointer
Exception: 'HH\Vector::fromItems' called dynamically without using a function pointer
Exception: 'HH\Vector::fromItems' called dynamically without using a function pointer
Exception: 'HH\Vector::fromItems' called dynamically without using a function pointer
Exception: 'HH\Vector::firstValue' called dynamically without using a function pointer
Exception: 'HH\Vector::fromItems' called dynamically without using a function pointer
Exception: 'array_map' called dynamically without using a function pointer
Exception: 'func2' called dynamically without using a function pointer
Exception: 'async_func2' called dynamically without using a function pointer
Exception: 'A::static_func2' called dynamically without using a function pointer
Exception: 'A::static_async_func2' called dynamically without using a function pointer
Exception: 'A::static_func2' called dynamically without using a function pointer
Exception: 'A::static_async_func2' called dynamically without using a function pointer
Exception: 'A::func2' called dynamically without using a function pointer
Exception: 'A::static_func2' called dynamically without using a function pointer
Exception: 'A::async_func2' called dynamically without using a function pointer
Exception: 'A::static_async_func2' called dynamically without using a function pointer
Exception: 'A::static_func2' called dynamically without using a function pointer
Exception: 'A::static_async_func2' called dynamically without using a function pointer
Exception: 'A::static_func2' called dynamically without using a function pointer
Exception: 'A::static_async_func2' called dynamically without using a function pointer
Exception: 'A::func2' called dynamically without using a function pointer
Exception: 'A::async_func2' called dynamically without using a function pointer
Exception: 'func2' called dynamically without using a function pointer
Exception: 'A::static_func2' called dynamically without using a function pointer
Exception: 'A::static_func2' called dynamically without using a function pointer
Exception: 'A::func2' called dynamically without using a function pointer
Exception: 'A::static_func2' called dynamically without using a function pointer
Exception: 'func2' called dynamically without using a function pointer
Exception: 'A::static_func2' called dynamically without using a function pointer
Exception: 'A::static_func2' called dynamically without using a function pointer
Exception: 'A::func2' called dynamically without using a function pointer
Exception: 'A::static_func2' called dynamically without using a function pointer
Exception: 'func2' called dynamically without using a function pointer
Exception: 'A::static_func2' called dynamically without using a function pointer
Exception: 'A::static_func2' called dynamically without using a function pointer
Exception: 'A::func2' called dynamically without using a function pointer
Exception: 'A::static_func2' called dynamically without using a function pointer
Exception: 'func2' called dynamically without using a function pointer
Exception: 'A::static_func2' called dynamically without using a function pointer
Exception: 'A::static_func2' called dynamically without using a function pointer
Exception: 'A::func2' called dynamically without using a function pointer
Exception: 'A::static_func2' called dynamically without using a function pointer
Exception: 'cmp2' called dynamically without using a function pointer
Exception: 'A::static_cmp2' called dynamically without using a function pointer
Exception: 'A::static_cmp2' called dynamically without using a function pointer
Exception: 'A::cmp2' called dynamically without using a function pointer
Exception: 'A::static_cmp2' called dynamically without using a function pointer